helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableNode.java

```diff
@@ -19,11 +19,7 @@
  * under the License.
  */
 
-import org.apache.helix.HelixException;
-import org.apache.helix.model.ClusterConfig;
-import org.apache.helix.model.InstanceConfig;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import static java.lang.Math.max;
```

Comment:
This import order is different from the default IDE config, I think.
@pkuwm Is it possible to add the import order in the style file? When do you plan to work on issue #349?

Reply:
It's auto-formatted as it is

Reply:
@jiajunwang I am working on the task with higher priority. But I can add the import order in the style file soon over the weekend

Reply:
@i3wangyi 's format is correct. I was using my homemade configure. @pkuwm Please add the import order like what @i3wangyi is doing here. Thanks.